test/clj_puppetdb/testutils/repl.clj

```diff
@@ -0,0 +1,5 @@
+(ns clj-puppetdb.testutils.repl
+  (:require [clojure.tools.namespace.repl :refer (refresh)]))
+
+(defn reset []
+  (refresh))
```

Comment:
It seems to work. It's really just a shortcut for refresh but it's nice to have something that works in our "usual" way.